### ISSUE TITLE
Why define the controller functions as methods on the global object?

### ISSUE THREAD

**Issue by ashkan-pm** [babak1368/MicroserviceSample#2]
https://github.com/babak1368/MicroserviceSample/blob/5682ed34757b3f06fcd66ee35dbf763dd01edf0f/saleService/controller/customerController.js#L5

If I'm not mistaken. These functions (handlers) will be assigned to methods on the global object. Why not just use a `const` before the name of the handler?

**Comment by babak1368** [comment]
Yeah, I should have const while I am encapsulating  the functions inside the customerController module.
`module.exports = {
    listHandler,
    createHandler,
    updateHandler,
    deleteHandler
}`
So, Will be changed and corrected .
